package.json

```diff
@@ -42,13 +42,14 @@
     "images-require-hook": "^1.0.3",
     "isomorphic-fetch": "^2.2.1",
     "memcached": "^2.2.2",
+    "moment": "^2.17.1",
     "nconf": "^0.8.4",
     "nconf-yaml": "^1.0.2",
     "normalize.css": "^5.0.0",
     "oauth": "^0.9.14",
     "openid": "^2.0.6",
-    "qs": "^6.3.0",
     "parse-github-url": "^0.3.2",
+    "qs": "^6.3.0",
```

Comment:
I wouldn't blame npm on that. I guess `qs` was added manually instead of using `npm --save` so it slipped a bit in sorting.